spec/requests/users_spec.rb

```diff
@@ -3,15 +3,30 @@
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/users/index'
+      get '/users/'
       expect(response).to have_http_status(:success)
     end
+
+    it 'returns http success' do
+      get '/users/:id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should include correct placeholder' do
+      get '/users/:id'
+      expect(response.body).to include('Here are the users')
+    end
```

Comment:
- [ ] Let's try to be descriptive in our tests and arrange them as they should be. `/users/:id` is a show route but you are defining its test under the `GET /index` description. Kindly remove these two tests from here. The correct route should be `/users/`

- [ ] Kindly add tests for rendering the template.

  ```
  expect(response).to render_template(:index)
  ```